spec/migrations/20171026114327_add_deleted_on_to_container_quota_and_items_spec.rb

```diff
@@ -0,0 +1,109 @@
+require_migration
+
+describe AddDeletedOnToContainerQuotaAndItems do
+  let(:container_quota_stub) { migration_stub(:ContainerQuota) }
+  let(:container_quota_item_stub) { migration_stub(:ContainerQuotaItem) }
+
+  let(:quota_data) do
+    {
+      :name             => "compute-resources",
+      # This is a real example, quota existed in openshift 2 month before
+      # provider was even added and refresh created DB record.
+      :created_on       => "2017-10-19 09:07:28 UTC",
+      :ems_created_on   => "2017-08-09 11:56:21 UTC",
+      :ems_ref          => "c2a9805e-7cf9-11e7-8ac6-001a4a162683",
+      :resource_version => "5553729",
+      # omitted ems_id, container_project_id.
+    }
+  end
+  let(:item_data) do
+    {
+      :resource       => "cpu",
+      :quota_desired  => 0.02,
+      :quota_enforced => 0.02,
+      :quota_observed => 0.0,
+    }
+  end
+
+  migration_context :up do
+    it "succeeds with zero container_quota_items" do
+      migrate
+    end
```

Comment:
This test is not necessary...every data migration has an automatic "empty tables" test.  https://github.com/ManageIQ/manageiq-schema/blob/master/spec/support/migration_helper.rb#L16

I'm not going to let this hold up the merge, as we can fix in a follow up PR.